package.json

```diff
@@ -3,13 +3,13 @@
     "aws-sdk": "^2.846.0",
     "bluebird": "^3.5.0",
     "lodash": "^4.17.20",
-    "proxy-agent": "^3.0.3",
+    "proxy-agent": "^4.0.1",
```

Comment:
fingers crossed, the major version bump won't introduce any breaking changes with how we use this module, but we should be cautious when updating the first few repos that use the aws lib and its use of proxy agent